emsdk.py

```diff
@@ -114,15 +114,16 @@ def exit_with_error(msg):
 UNIX = (MACOS or LINUX)
 
 
-# Pick which shell of 4 shells to use
+# Pick which shell of 5 shells to use
```

Comment:
Maybe just `# Pick which shell to use`?

Reply:
I wonder if we should add some basic CI testing for other shells?

Reply:
👌 I agree with you.